api/utils/hysds_util.py

```diff
@@ -633,17 +633,22 @@ def get_mozart_jobs(username, page_size=10, offset=0):
         :return:
         """
     params = dict()
-    params["page_size"] = page_size
-    params["offset"] = offset  # this is specifies the offset
+    logging.info("hysds utils")
+    for key, value in kwargs.items():
+        logging.info("key: {}, value: {}".format(key, value))
+        params[key] = value
 
     session = requests.Session()
     session.verify = False
 
+    logging.info("Job params: {}".format(params))
```

Comment:
Are these multiple logging statements for the same items being used for dev debugging ? 
Is there a need for this log line given that you are logging in the loop and then again on line 651 ?

Reply:
Changed log level to 'debug'